crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -4925,8 +4919,11 @@ impl<'a> Checker<'a> {
                 // Collect all unused imports by location. (Multiple unused imports at the same
                 // location indicates an `import from`.)
                 type UnusedImport<'a> = (&'a str, &'a Range);
-                type BindingContext<'a, 'b> =
-                    (&'a RefEquality<'b, Stmt>, Option<&'a RefEquality<'b, Stmt>>);
+                type BindingContext<'a, 'b> = (
+                    &'a RefEquality<'b, Stmt>,
+                    Option<&'a RefEquality<'b, Stmt>>,
+                    Exceptions,
+                );
```

Comment:
Nit: Can we add some documentation about the different tuple fields or maybe it's best to create a struct at this point with named fields to improve readability

Reply:
Shoot, this got auto-merged. Yes I'll do a pass on this.